docs/HowToConnectSQLDatabase.md

```diff
@@ -0,0 +1,43 @@
+# Connecting to a SQL Database
+The procedures for connecting to a SQL database are outlined in this article.
+
+## Active Directory Administrator for SQL Server
+
+The user must set themselves as SQL Server Active Directory Admin when they connect to the database for the first time. 
+
+The actions to do are listed below for it.
+*	Access the SQL Server
+*	Click on Settings -> Microsoft Entra ID -> Set Admin
+![alt text](images/HowToConnectSQLDatabase/entraid-admin.png "Entra Admin")
+*	Choose the User Account
+![alt text](images/HowToConnectSQLDatabase/choose-user-account.png "Entra Admin")
+*	Press the "Save" button.
+![alt text](images/HowToConnectSQLDatabase/save-user-admin-account.png "Entra Admin")
+
+At this point, the user can connect to the database using `Active Directory authentication`.
+
+Create a new user in the database and give them the role `db_owner` after connecting to the database.
+
+The script for the same can be found below: 
+```
+CREATE USER [user-name] FROM EXTERNAL PROVIDER; 
+ALTER ROLE db_owner ADD MEMBER [user-name];
+```
+
+Example query
+```
+CREATE USER [v-test@microsoft.com] FROM EXTERNAL PROVIDER;
+ALTER ROLE db_owner ADD MEMBER [v-test@microsoft.com]; 
+```
+
+Revert the SQL Server Active directory admin user changes once the user has been created, that is, set the `OSS FHIR Server(App service)` user assigned identity back to SQL Server admin.
```

Comment:
@SergeyGaluzo is this the recommended approach?

Reply:
@brendankowitz What problem are we trying to solve? Is something not working with managed identities?

Reply:
I would suggest different approach. FHIR SQL server does not need FHIR Server UAMI as admin for FHIR server to work, it is just hard to setup by the pipeline. Therefore, I usually "downgrade" UAMI to db owner after FHIR server is setup. This contains similar steps to create login for UAMI and corresponding database user. Setting up user access becomes a separate issue. In certain cases, it could remain admin...

Reply:
@brendankowitz: Please let us know if you suggest us to update the pipeline and documentation with an alternative approach, and if you'd like us to include instructions on providing user access to the database which will be the separate issue as mentioned.